src/minimap.cpp

```diff
@@ -550,13 +554,12 @@ void Minimap::updateActiveMarkers()
 	video::IImage *minimap_mask = data->minimap_shape_round ?
 		data->minimap_mask_round : data->minimap_mask_square;
 
-	std::list<Nametag *> *nametags = client->getCamera()->getNametags();
+	const std::list<Nametag *> &nametags = client->getCamera()->getNametags();
 
 	m_active_markers.clear();
 
-	for (std::list<Nametag *>::const_iterator
-			i = nametags->begin();
-			i != nametags->end(); ++i) {
+	for (std::list<Nametag *>::const_iterator i = nametags.begin();
+			i != nametags.end(); ++i) {
 		Nametag *nametag = *i;
 		v3s16 pos = floatToInt(nametag->parent_node->getPosition() +
```

Comment:
i don't override it, i just make iterator constent to be consistent with the list